src/schemathesis/utils.py

```diff
@@ -30,3 +30,8 @@ def force_tuple(item: Filter) -> Union[List, Set, Tuple]:
     if not isinstance(item, (list, set, tuple)):
         return (item,)
     return item
+
+
+def dict_true_values(**kwargs):
+    """Create dict with given kwargs while skipping items where bool(value) evaluates to False."""
+    return {key: kwargs[key] for key in kwargs if bool(kwargs[key])}
```

Comment:
We can avoid extra `__getitem__` by using `.items()`:

```
{key: value for key, value in kwargs.items() if bool(value)}
```

What do you think?

Reply:
Yep, I though we are creating list of tuples by calling `.items()` but now I'm realizing we are not in python 2 anymore :-D Will fix that.